ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/TransactionWithMetadata.java

```diff
@@ -1,6 +1,5 @@
 /*
- *
- * Copyright ConsenSys AG.
+ * Copyright contributors to Hyperledger Besu.
```

Comment:
some of these are changing from consensys to hl but I don't think that's a huge problem

Reply:
thanks for catching these, they missed the regular expression in earlier PRs.